config/ide-helper.php

```diff
@@ -111,6 +111,21 @@
 
     ),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Models hooks
+    |--------------------------------------------------------------------------
+    |
+    | Define which hook classes you want to run for models to add custom information
+    |
+    | Hooks should implement Contracts\ModelHookInterface.
+    |
+    */
+
+    'model_hooks' => array(
+
```

Comment:
Maybe an commented pseudo-example to be clear this expects something like `// App\Services\IdeHelper\MyHook::class` or so?

Reply:
I should probably also update the readme to include a bit about using model hooks.

Reply:
Now that you mention it, good idea 😄